funnel/models/project.py

```diff
@@ -374,7 +379,7 @@ def calendar_weeks(self):
             SELECT DATE_TRUNC('day', "start_at" AT TIME ZONE :timezone) AS date, COUNT(*) AS count
             FROM "session" WHERE "project_id" = :project_id AND "start_at" IS NOT NULL AND "end_at" IS NOT NULL
             GROUP BY date ORDER BY date;
-            ''')).params(timezone=self.timezone.zone, project_id=self.id)
+            ''')).params(timezone=self.timezone.zone, project_id=self.id).all()
```

Comment:
Since you are modifying the list, don't use `.all()`. Use `list(...)` instead. It ensures you have a local list and aren't mutating someone else's list.